airbyte-integrations/connectors/source-stripe/unit_tests/integration/test_authorizations.py

```diff
@@ -243,7 +203,7 @@ def test_given_no_state_when_read_then_use_authorizations_endpoint(self, http_mo
         output = self._read(_config().with_start_date(_A_START_DATE), _NO_STATE)
         most_recent_state = output.most_recent_state
         assert most_recent_state.stream_descriptor == StreamDescriptor(name=_STREAM_NAME)
-        assert most_recent_state.stream_state == AirbyteStateBlob(updated=cursor_value)
+        assert most_recent_state.stream_state.__dict__ == {"updated": str(cursor_value)}
```

Comment:
The state changes to a string as we have seen for other sources. I think we wanted to validate if this change was backward compatbible. @pnilan have we confirmed that?